frameworks/flaml/exec.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import warnings
+import pandas as pd
+
+from amlb.utils import Timer
+
+from flaml import AutoML, __version__
+
+from frameworks.shared.callee import call_run, result, save_metadata
+
+log = logging.getLogger(__name__)
+
+
+def run(dataset, config):
+    log.info("\n**** FLAML ****\n")
+    save_metadata(config, version=__version__)
+    time_budget = config.max_runtime_seconds
+    n_jobs = config.framework_params.get('_n_jobs', config.cores)
+
+    print("Running FLAML with {} number of cores".format(config.cores))
+
+    is_classification = config.type == 'classification'
+    column_names, _ = zip(*dataset.columns)
+    column_types = dict(dataset.columns)
+    train = pd.DataFrame(dataset.train.data, columns=column_names).astype(
+        column_types, copy=False)
+    label = dataset.target.name
+    X_train = y_train = None
+    test = pd.DataFrame(dataset.test.data, columns=column_names).astype(
+        column_types, copy=False)
+    X_test = test.drop(columns=label)
+    y_test = test[label]
```

Comment:
sorry that you have to do that.
I'll work very soon on switching the `amlb` app to `pandas` instead of `numpy`: this should simplify many things, bring support for `string` columns for frameworks that support them, and also ensure that all frameworks get the same data and metadata.